samples/snippets/src/main/java/com/example/bigquerystorage/WriteToDefaultStream.java

```diff
@@ -26,14 +29,21 @@
 import com.google.cloud.bigquery.storage.v1.JsonStreamWriter;
 import com.google.cloud.bigquery.storage.v1.TableName;
 import com.google.cloud.bigquery.storage.v1.TableSchema;
+import com.google.common.util.concurrent.MoreExecutors;
 import com.google.protobuf.Descriptors.DescriptorValidationException;
 import java.io.IOException;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Phaser;
 import org.json.JSONArray;
 import org.json.JSONObject;
 
 public class WriteToDefaultStream {
 
+  private final Phaser inflightRequestPhaser = new Phaser(1);
```

Comment:
We don't prefer having anything out here like this, they should go inside either the driver method (`runWriteToDefaultStream`) or the body of the sample (`writeToDefaultStream`). Also, we need comment (explanation) around why we are using `inflightRequestPhaser`.

Reply:
Added a comment about the Phaser (which we need in order to wait for all responses before closing the stream)

Discussed the variables offline: the Write API is a special case where we need a stateful client and want to make explicit that users should keep the writer around for as long as possible.